specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/stable/2021-04-01/Disks.json

```diff
@@ -0,0 +1,441 @@
+    "title": "ComputeDiskAdminManagementClient",
+    "description": "The Admin Compute Disk Management Client."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Compute.Admin/locations/{location}/disks": {
+      "get": {
+        "x-ms-examples": {
+          "Returns a list of disks.": {
+            "$ref": "./examples/Disks/List.json"
+          }
+        },
+        "tags": [
+          "Disks"
+        ],
+        "operationId": "Disks_List",
+        "description": "Returns a list of disks.",
+        "parameters": [
+          {
+            "$ref": "Disks.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "Disks.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "Disks.json#/parameters/UserSubscriptionIdParameter"
+          },
+          {
+            "$ref": "Disks.json#/parameters/DiskStateParameter"
+          },
+          {
+            "$ref": "Disks.json#/parameters/SourceSharePathParameter"
+          },
+          {
+            "$ref": "Disks.json#/parameters/SourceScaleUnitParameter"
+          },
+          {
+            "$ref": "Disks.json#/parameters/SourceVolumeLabelParameter"
+          },
+          {
+            "$ref": "Disks.json#/parameters/DiskCountParameter"
+          },
+          {
+            "$ref": "Disks.json#/parameters/DiskStartParameter"
+          },
+          {
+            "$ref": "Disks.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- The list of disks has been returned.",
+            "schema": {
+              "$ref": "Disks.json#/definitions/DiskList"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Compute.Admin/locations/{location}/disks/{diskId}": {
+      "get": {
+        "x-ms-examples": {
+          "Returns the requested disk.": {
+            "$ref": "./examples/Disks/Get.json"
+          }
+        },
+        "tags": [
+          "Disks"
+        ],
+        "operationId": "Disks_Get",
+        "description": "Returns the disk.",
+        "parameters": [
+          {
+            "$ref": "Disks.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "Disks.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "Disks.json#/parameters/DiskIdParameter"
+          },
+          {
+            "$ref": "Disks.json#/parameters/DiskSizeDetailParameter"
+          },
+          {
+            "$ref": "Disks.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "default": {
```

Comment:
>"default" [](http://example.com/codeflow?start=10&length=9)

Same as my other comment, make sure to include default responses in all your APIs.